magda-web-client/src/Components/Common/DataPreviewMapOpenInNationalMapButton.tsx

```diff
@@ -67,7 +68,52 @@ class DataPreviewMapOpenInNationalMapButton extends Component<PropsType> {
                 ]
             };
         } else {
+            const dataUrl = distribution?.downloadURL
+                ? distribution.downloadURL
+                : distribution.accessURL;
+
+            let layers: any = undefined;
             const id = "data.gov.au-postMessage-" + distribution?.identifier;
+            if (dataUrl) {
+                const queries = URI.parseQuery(dataUrl);
+                layers = queries["layers"];
+                if (typeof layers !== "string") {
+                    layers = queries["LAYERS"];
+                }
+
+                if (typeof layers !== "string") {
+                    layers = queries["Layers"];
+                }
+            }
+
+            const type =
+                distribution?.format.toLowerCase() === "wms"
+                    ? layers
+                        ? "wms"
+                        : "wms-group"
+                    : undefined;
+
+            const terriaAspect: any = type
+                ? {
+                      aspects: {
+                          terria: {
+                              definition: {
+                                  name:
+                                      distribution?.title +
+                                      " (Opened by data.gov.au)",
```

Comment:
We can't hardcode `(Opened by data.gov.au)` --- the same version is used by many clients e.g. data.NSW
And they care about branding.
If we have to mentioned the current APP name, it can be retrieved via content API stored `lang/en/global/appName` value --- it's already retrieved and store in the reduex state I think

Reply:
Replaced with generic terms `(Added by external application)`.